DrawerController/DrawerController.swift

```diff
@@ -358,7 +358,7 @@ open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
                     ss.didFinishAnimate(side: .none, percent: 0.0)
                     
                     ss.isAnimating = false
-                    ss.delegate?.drawerDidCloseSide?(drawerController: ss)
+                    ss.delegate?.drawerDidCloseSide?(drawerController: ss, side: ss.drawerSide)
```

Comment:
If you do not write oldValue, there is likely to be some variation in the middle.

```swift
let oldDrawerSide = drawerSide

delegate?.drawerWillCloseSide?(drawerController: self, side: oldDrawerSide)

...

ss.delegate?.drawerDidCloseSide?(drawerController: ss, side: oldDrawerSide)

...
```